README.md

```diff
@@ -52,7 +52,7 @@ Alternatively you can include the script like so:
 InPlayer.Account.signIn({
     email: 'test@test.com',
     password: '12345678',
-    cliendId: 'b0899d7f-66da-40fc-8eeb-36cad735589c',
+    clientId: 'b0899d7f-66da-40fc-8eeb-36cad735589c',
```

Comment:
The same typo is in several other files as well: `examples/v2.html` and `examples/v3.html`

Reply:
fixed on new PR https://github.com/inplayer-org/inplayer.js/pull/810